src/OutputHash.js

```diff
@@ -173,6 +223,10 @@ OutputHash.prototype.apply = function apply(compiler) {
 
         // Reuses webpack options
         hashFn = input => {
+            // generate hash of content without sourcemap
+            const mapCommentReg = /\s\/[\/|*]#\s?sourceMappingURL=.*.map(\s\*\/)?/g;
```

Comment:
Shouldn't this regex match only on the beginning of the line? Also, can there be more than one mapComment?